tools/create_release_files.sh

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Quick-and-dirty automation of making maven and binary releases. Not robust at all.
+# Publishes releases to Maven and packages/copies binary release artifacts.
+# Expects to be run in a totally empty directory.
+#
+
+#
+#  NOTE: The code in this file is based on code from the Apache Spark
+#  project, licensed under the Apache License v 2.0
+#
+#  https://github.com/apache/spark/blob/master/dev/create-release/create-release.sh
```

Comment:
hmm...do you know by chance which version you took it from?

Otherwise i would replace it with `https://github.com/apache/spark/tree/branch-1.0/dev/create-release`